src/versions/validation/index.js

```diff
@@ -112,7 +112,11 @@ const middleware = () => {
       return setImmediate(next)
     }
 
-    const { schema } = validationDefinition
+    const { schema: schemaObjOrFunction } = validationDefinition
```

Comment:
In your commit message, you say:

> This solves definition objects order in validation files.

Could you elaborate with an example?

Reply:
Yes that wasn't clear enough.

I wanted to define validation schemas difference in anti-chronological order.
Without this modification, it wasn't possible to reference an object that is define further in the file.

Example: `schemas['2020-08-10'].list` references `schemas['2019-05-20'].list`
https://github.com/stelace/stelace/blob/6b5a6ef5619a71b4f7ca175c46c4883c83708067/src/versions/validation/event.js#L68-L78
https://github.com/stelace/stelace/blob/6b5a6ef5619a71b4f7ca175c46c4883c83708067/src/versions/validation/event.js#L113-L134

Now validation schema can be a function that returns the validation schema only when needed (at the endpoint hit).

Reply:
Nice, this may be a performance hit though, unless you use singleton constants.

Reply:
Indeed, I'll update the code to avoid this performance issue.